.github/workflows/develop-ci.yml

```diff
@@ -53,3 +53,19 @@ jobs:
 
       - name: Test yul tests
         run: make test_yul
+
+      - name: Generate benchmarks for tests
+        run: |
+          mkdir -p ./benchmark/stats
+          make test_bench
```

Comment:
I wouldn't call it `test_bench`, we don't test anything, we just benchmark, but it doesn't matter all that much